src/Tools/dotnet-monitor/CollectionRules/ExecuteAction.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.Diagnostics.NETCore.Client;
```

Comment:
Add copyright header.

Reply:
Also, please add a Actions folder under src/Tools/dotnet-monitor/CollectionRules and move this file to that folder.